### PR TITLE
[SMP] Establish experiment naming prefix convention for Quality Gates

### PR DESCRIPTION
**Pull request by scottopell**
### What does this PR do?

Duplicates the two existing quality-gate experiments and prefixes the experiment name with `quality_gate_`

### Motivation

Some SMP experiments are designed to be the most representative of overall Datadog Agent behavior, these will have strict upper bounds and will have been audited in more depth than others.

These "Quality Gate" experiments will be identified by this `quality_gate_` prefix.

### Describe how to test/QA your changes
n/a

### CODE REVIEW

File: test/regression/cases/quality_gate_idle_all_features/experiment.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Agent 'all features enabled' idle experiment. Represents an agent install with
+# all sub-agents enabled in configuration and no active workload.
```
Comment (GeorgeHahn) [inline test/regression/cases/quality_gate_idle_all_features/experiment.yaml:2]: What do you think about adding a blurb about quality gates and a link to our docs in these?

**Review by GeorgeHahn — APPROVED** [review]

File: test/regression/cases/quality_gate_idle_all_features/experiment.yaml (hunk 1 above)
Comment (scottopell) [reply test/regression/cases/quality_gate_idle_all_features/experiment.yaml:2]: Yes, I plan to add this once the corresponding confluence docs exist

Comment (GeorgeHahn) [reply test/regression/cases/quality_gate_idle_all_features/experiment.yaml:2]: 👀 🙈

**Pull request merged**